dev-scripts/test-all.sh

```diff
@@ -19,20 +19,24 @@ cat  <<END
 This script should run all of the tests that CI will run. (It'll need to be kept up to date though)
 Options:
   -s: Skip emulator tests
+  -u: Set Unbundled AAOS path
 Pre-requisites: 
-    Must be run on a system that can run emulators
-    Have \$ORG_GRADLE_PROJECT_unbundledAAOSDir set to your AAOS Unbundled repo
     Have \$FIGMA_ACCESS_TOKEN set to your actual Figma token
+    To run the full suite your system must be able to run emulator tests.
+      Use the -s argument to skip emulator tests.
 END
 }
 
 OPTIND=1         # Reset in case getopts has been used previously in the shell.
 run_emulator_tests=1
-while getopts "s" opt; do
+while getopts "su:" opt; do
```

Comment:
What's the colon here for?

Reply:
It's part of getopts, it specifies that this option takes an argument and isn't just a flag.